test/unit/org/apache/cassandra/cql3/PstmtPersistenceTest.java

```diff
@@ -178,7 +400,6 @@ private MD5Digest prepareStatement(String stmt, ClientState clientState)
 
     private MD5Digest prepareStatement(String stmt, String keyspace, String table, ClientState clientState)
     {
-        System.out.println(stmt + String.format(stmt, keyspace + "." + table));
```

Comment:
Not sure how that println got in there but it's been there for 4 years apparently, figure it's ok to remove.

Reply:
That's amazing lol. Good catch